trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -205,6 +209,39 @@ const sendManualBankDecisionEmail = async (amendmentVariables) => {
   }
 };
 
+/**
+ * Evaluated whether facility amendment is eligible
+ * for ACBS interaction based on myriads of conditions.
+ * This function evaluated across all amendment types.
+ * @param {Object} amendment Facility amendments object
+ */
+const canSendToAcbs = (amendment) => {
+  // Ensure at least one of the attribute has been amended
+  const hasBeenAmended = amendment.changeCoverEndDate || amendment.changeFacilityValue;
+  // Amendment status is marked as `Completed`
+  const completed = amendment.status === AMENDMENT_STATUS.COMPLETED;
+  // Amendment has been submitted by PIM team
+  const PIM = Boolean(amendment.submittedByPim);
+  // Manual amendment verification
+  const manual = Boolean(amendment.requireUkefApproval) && Boolean(amendment.bankDecision);
```

Comment:
we shouldn't need the typecast because `requireUkefApproval` is always boolean. If you need to know whether an amendment was submitted back to the bank, then I think there is a specific flag that you can use for that - which would be a boolean flag already

Reply:
if we just need to check whether an amendment is manual, then we only need to check the `requireUkefApproval` property.

Reply:
I remember you were working on this change the other day. did you push it?